accumulo-store/src/main/java/gaffer/accumulostore/utils/Constants.java

```diff
@@ -43,32 +46,57 @@
     public static final String BLOOM_FILTER_CHARSET = "ISO-8859-1";
 
     // Iterator priorities
-    public static final int AGE_OFF_ITERATOR_PRIORITY = 10; // Applied during major compactions, minor compactions and scans.
-    public static final int AGGREGATOR_ITERATOR_PRIORITY = 20; // Applied during major compactions, minor compactions and scans.
-    public static final int BLOOM_FILTER_ITERATOR_PRIORITY = 31; // Applied only during scans.
-    public static final int EDGE_ENTITY_DIRECTED_FILTER_ITERATOR_PRIORITY = 32; // Applied only during scans.
-    public static final int ELEMENT_FILTER_ITERATOR_PRIORITY = 33; // Applied only during scans.
-    public static final int QUERY_TIME_AGGREGATOR_PRIORITY = 34; // Applied only during scans.
-    public static final int TRANSFORM_PRIORITY = 50; // Applied only during scans.
+    public static final int AGE_OFF_ITERATOR_PRIORITY = 10; // Applied during
```

Comment:
Looks like your ide reformatted your comments here and formatted them strangely. Perhaps put the comment on 1 line or on the line before or as a javadoc